docs/cql/types.rst

```diff
@@ -334,6 +334,10 @@ Further, maps support:
 
   Note that for removing multiple elements in a ``map``, you remove from it a ``set`` of keys.
 
+- Selecting one element::
+
+    SELECT favs['fruit'] FROM users WHERE id = 'jsmith';
+
 Lastly, TTLs are allowed for both ``INSERT`` and ``UPDATE``, but in both cases, the TTL set only applies to the newly
 inserted/updated elements. In other words::
 
```

Comment:
The grammar is repeated in select.rst (there's also a reference in update.rst but it need not be updated).

Reply:
This documentation change only the case of maps, it doesn't cover what happens to lists or sets (it's particularly non-obvious for sets!). I don't insist we need to fix the documentation right now (I'm guessing it is missing a lot of other things too), but if you know where to change, maybe it's easier to just change it right now instead of remembering to go back to it later.

Reply:
I added similar examples for list and set
I'm a little hesitant to add it to the grammar documentation because I don't want to make it more complicated, and I see that similarly it doesn't have UDT field selection, and also Cassandra doesn't document it

Reply:
I don't want to adopt the precedent that because X is undocumented, then when we add Y it's also undocumented. Anything we add should be fully documented. No need to retroactively document stuff, just don't create new gaps.

Reply:
I added the subscript and field selection to the grammar documentation